frontends/web/src/components/balance/balance.test.tsx

```diff
@@ -0,0 +1,58 @@
+/**
+ * Copyright 2022 Shift Crypto AG
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { render } from '@testing-library/react';
+import { IBalance } from '../../api/account';
+import { Balance } from './balance';
+
+describe('components/balance/balance', () => {
+  it('renders balance properly', () => {
+    const MOCK_BALANCE: IBalance = {
+      hasAvailable: true,
+      hasIncoming: true,
+      available: {
+        amount: '0.005',
+        unit: 'BTC'
+      },
+      incoming: {
+        amount: '0.003',
+        unit: 'BTC',
+        conversions: {
+          BTC: '0.003',
+          AUD: '512',
+          BRL: '512',
+          CAD: '512',
+          CHF: '512',
+          CNY: '512',
+          EUR: '512',
+          GBP: '512',
+          HKD: '512',
+          ILS: '512',
+          JPY: '512',
+          KRW: '512',
+          NOK: '512',
+          RUB: '512',
+          SEK: '512',
+          SGD: '512',
+          USD: '512',
+        }
+      }
+    };
+    const { getByTestId } = render(<Balance balance={MOCK_BALANCE} />);
+    expect(getByTestId('availableBalance')).toHaveTextContent('0.005BTC');
+    expect(getByTestId('incomingBalance')).toHaveTextContent('+0.003 BTC / 512 USD');
+  });
+});
```

Comment:
Hm, good catch! Unfort. using `useTranslation` wont solve this issue as some components, such as "balance" imports component(s) which imports another component(s).. etc that use translation. See below for details. I think in `language.test.tsx` we could use it because I believe we don't import  anything with translation in `language.tsx`. The only component using translation is **that component itself**.

**Anyhow, to fix this warning issue we could render the component (in this case `Balance`), inside a `<I18nextProvider/>`**. I will make this change! 🙂 



FYI, this is the error when mocking `useTranslation` in `Balance`:
```js
    TypeError: (0 , _translate.translate)(...) is not a function

      129 | }
      130 |
    > 131 | const HOC = translate()(share<SharedProps, TranslateProps>(store)(Guide));
          |             ^
      132 | export { HOC as Guide };
```
The trace:

```js
    at Object.<anonymous> (src/components/guide/guide.tsx:131:13)
      at Object.<anonymous> (src/components/layout/header.tsx:20:1)
      at Object.<anonymous> (src/components/layout/index.tsx:18:1)
      at Object.<anonymous> (src/components/view/view.tsx:19:1)
      at Object.<anonymous> (src/components/alert/Alert.tsx:21:1)
      at Object.<anonymous> (src/utils/request.ts:19:1)
      at Object.<anonymous> (src/utils/config.js:17:1)
      at Object.<anonymous> (src/components/rates/rates.tsx:22:1)
      at Object.<anonymous> (src/components/balance/balance.tsx:20:1)
      at Object.<anonymous> (src/components/balance/balance.test.tsx:19:1)
      at TestScheduler.scheduleTests (node_modules/@jest/core/build/TestScheduler.js:333:13)
      at runJest (node_modules/@jest/core/build/runJest.js:404:19)
```